slave.mk

```diff
@@ -1210,21 +1210,23 @@ $(DOCKER_LOAD_TARGETS) : $(TARGET_PATH)/%.gz-load : .platform docker-start $$(TA
 ## Installers
 ###############################################################################
 
-ifeq ($(ENABLE_RFS_SPLIT_BUILD),y)
+
 define rfs_get_installer_dependencies
 $(1)__$($(1)_MACHINE)__rfs.squashfs $(if $($(1)_DEPENDENT_MACHINE),$(1)__$($(1)_DEPENDENT_MACHINE)__rfs.squashfs)
 endef
 
 $(foreach installer, $(SONIC_INSTALLERS), $(eval $(installer)_RFS_DEPENDS=$(call rfs_get_installer_dependencies,$(installer))))
-
 SONIC_RFS_TARGETS= $(foreach installer, $(SONIC_INSTALLERS), $(call rfs_get_installer_dependencies,$(installer)))
+
+ifeq ($(ENABLE_RFS_SPLIT_BUILD),y)
```

Comment:
Can you split the build_debian.sh into two separate files?
1. build_debian_rootfs.sh  => which contains all the standard squashfs packages
2. build_debian_sonic.sh => Includes all the sonic packages into existing squashfs and creates the dockerfs.

Reply:
I'm not sure I understand why splitting the build_debian.sh is better. Also, with the current design, the feature is optional and configurable (via ENABLE_RFS_SPLIT_BUILD). If we split the build_debian.sh, the current way to build will no longer be available. 

Can you please help to understand why the _if..else_ is better avoided?